shell/platform/darwin/ios/framework/Source/accessibility_bridge.h

```diff
@@ -139,6 +145,7 @@ class AccessibilityBridge final {
 
   UIView* view_;
   PlatformViewIOS* platform_view_;
+  FlutterPlatformViewsController* flutter_platform_views_controller_;
```

Comment:
`platform_views_controller` for consistency with elsewhere.

Reply:
done